matter/mbedtls/repo/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Classify all '.function' files as C for syntax highlighting purposes
```

Comment:
pretty sure we don't need that since it is not a git repo in this context

Reply:
resolve